### ISSUE TITLE
[DOC] - wandb tip for multi-processing

### ISSUE THREAD

**Issue by srygaard** [mila-iqia/mila-docs#420]
### 🔗 Concerned Page:

https://docs.mila.quebec/userguides/wandb/#diagnose-training-bottlenecks

### 🛠 Nature of the issue:
- [ ] Technical Error (e.g., broken command, dead link)
- [ ] Outdated Content (e.g., old software version, deprecated policy)
- [ ] Lack of Clarity (e.g., confusing explanation, missing examples)
- [x] Suggestion (e.g., new guide needed, additional tip)

### 📝 Description:

The section is about diagnosing training bottlenecks. For users looking at CPU bound parallel data loader bottlenecks, they may find the following setting and limitation of `wandb` important:

https://docs.wandb.ai/models/ref/python/experiments/settings#:~:text=x_stats_track_process_tree%20(bool)%3A%20Monitor,disabled%20by%20default

> **x_stats_track_process_tree** (bool): 
> Monitor the entire process tree for resource usage, starting from x_stats_pid. When True, the system monitor aggregates the RSS, CPU%, and thread count from the process with PID x_stats_pid and all of its descendants. 
> This can have a performance overhead and is disabled by default.

PyTorch's `DataLoader(num_workers=0)` is often configured with `num_workers` values greater than 0 ([docs](https://docs.pytorch.org/docs/2.12/data.html#multi-process-data-loading)). These additional python subprocesses will be spawned but not tracked by `wandb`. 

In practice, jobs with many `DataLoader` workers may show much lower CPU utilization in `wandb` than the actual usage.

### 💡 Suggested Solution (Optional):

A tip stating that multi-process monitoring is disabled by default in `wandb` might be all that's necessary to avoid misinterpreting results.

### ⚡ Impact on Research:

- [ ] Blocker (I cannot complete my work without this info)
- [ ] Important (High risk of error or significant time loss)
- [x] Minor (Reading comfort, cosmetic/typo)

**Comment by satyaog** [comment]
Hey @srygaard ! we've just added a note in the guide : #423 . Let us know what you think and if that would cover the need explained here

**Comment by srygaard** [comment]
Looks good to me. Thank you @satyaog for the update.
